src/server/game/Entities/Unit/CharmInfo.cpp

```diff
@@ -88,6 +88,7 @@ void CharmInfo::InitPossessCreateSpells()
             case 40281: // Crashin' Thrashin' Racer
             case 23109: // Vengeful Spirit
             case 25653: // Power of the Blue Flight
+            case 21949: // Fel Reaver Sentinel
```

Comment:
Should no longer be needed with https://github.com/azerothcore/azerothcore-wotlk/commit/7abbca0f5b4ab3a15d78db7ec40e3b4358bed7da.

Reply:
yep, that fixes it